### ISSUE TITLE
Changes in 1.0.1 seems to have broken my logger

### ISSUE THREAD

**Issue by gitblit** [cybercoder-naj/logestic#20]
Given this logger...

```javascript
import { Logestic, chalk } from "logestic";

// exports a Logestic plugin instance
export const logger = new Logestic({ showLevel: true })
  .use(["time", "method", "path", "status", "duration"])
  .format({
    // onRequest({ time, method, path }) {
    //   const grayTime = chalk.gray(`${time!.toISOString()}`);
    //   const methodPath = chalk.cyan(`${method} ${path}`);

    //   return `[${grayTime}] 🟢 ${methodPath}`;
    // },
    onSuccess({ time, method, path, status, duration }) {
      const grayTime = chalk.gray(`${time!.toISOString()}`);
      const methodPath = chalk.cyan(`${method} ${path}`);
      let statusColor = chalk.white;

      if (200 <= status && status < 300) statusColor = chalk.green;
      if (400 <= status && status < 500) statusColor = chalk.yellow;
      if (500 <= status) statusColor = chalk.red;

      const durationInMsecs = chalk.magenta(`${(duration ?? 0n) / 1000n}`);

      return `[${grayTime}] 🛑 ${methodPath} | ${statusColor(status)} | ${durationInMsecs} ms\n`;
    },
    onFailure({ request, error, code, datetime }) {
      const log = chalk.red(`[${datetime.toISOString()}]: ${request.method} ${request.url} | ERROR | ${code}`);
      return `${log}\n${error}`;
    },
});
```

I get undefined errors on `time`. If I replace the `time` reference in the string interpolation with `(new Date())` then I get `undefined` on every argument in the `onSuccess` console logs. I have determined that this behavior change was introduced in 1.0.1.

**Comment by cybercoder-naj** [comment]
Very tiny error. My bad. Should be fixed in `v1.1.1`

**Comment by gitblit** [comment]
Looks good, thank you.
